core/chainlink-sgx.Dockerfile

```diff
@@ -1,19 +1,39 @@
 # Build Chainlink with SGX
-FROM smartcontract/builder:1.0.20 as builder
+FROM smartcontract/builder:1.0.25 as builder
 
 # Have to reintroduce ENV vars from builder image
 ENV PATH /root/.cargo/bin:/go/bin:/usr/local/go/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/opt/sgxsdk/bin:/opt/sgxsdk/bin/x64
 ENV LD_LIBRARY_PATH /opt/sgxsdk/sdk_libs
 ENV SGX_SDK /opt/sgxsdk
 
+WORKDIR /chainlink
```

Comment:
Did we want to use standard filesystem hierarchy here? Something like `/usr/local/src/chainlink`? It would keep it consistent with what we've done with other dockerfiles

Reply:
I'd change that in a separate PR if it's desirable, this is just a cleanup.